material-overrides/home.html

```diff
@@ -16,57 +16,51 @@
         </div>
       </div>
 
       <div class="feature-gradient">
         <div class="feature-section">
           <div class="feature-intro">
-            <h2>A network governed by a rebelliously innovative community</h2>
-            <p>Polkadot is a blockchain network of networks designed to challenge our assumptions, directed and governed by those who hold the DOT token. By holding DOT, you are part of the largest and most decentralized decision-making group in the world.</p>
+            <h2>Your complete guide to building on Polkadot</h2>
+            <p>Polkadot's enhanced developer docs provide all the guidance and resources you need to launch and scale your projects within the Polkadot ecosystem.</p>
           </div>
           <div class="featured-content">
-            <a class="feature" href="#">
+            <a class="feature" href="develop/blockchains/get-started/">
               <div class="column">
-                <h3>DOT Token Basics</h3>
-                <p> What’s in a DOT? Dive into the utility token that powers the Polkadot ecosystem.</p>
+                <h3>Start building blockchains</h3>
+                <p>Kickstart your journey with the foundational resources like the fundamentals of blockchain development, getting started with the Polkadot SDK, and building and deploying blockchains.</p>
               </div>
               <div class="column">
                 <span class="md-icon">{% include ".icons/octicons/arrow-up-right-16.svg" %}</span>
               </div>
             </a>
-            <a class="feature" href="#">
+            <a class="feature" href="develop/smart-contracts/">
               <div class="column">
-                <h3>DOT Token Basics</h3>
-                <p> What’s in a DOT? Dive into the utility token that powers the Polkadot ecosystem.</p>
+                <h3>Smart contract development</h3>
+                <p>Explore the smart contract landscape in the Polkadot ecosystem. Access tools and guides to build with ink!, Polkadot's native smart contract language, or work within EVM environments.</p>
               </div>
               <div class="column">
                 <span class="md-icon">{% include ".icons/octicons/arrow-up-right-16.svg" %}</span>
               </div>
             </a>
-            <a class="feature" href="#">
+            <a class="feature" href="polkadot-protocol/architecture/">
               <div class="column">
-                <h3>DOT Token Basics</h3>
-                <p> What’s in a DOT? Dive into the utility token that powers the Polkadot ecosystem.</p>
+                <h3>Polkadot architecture</h3>
+                <p>Unlock the potential of Polkadot's multi-chain architecture. From an overview of Polkadot to detailed insights on parachains and system chains, gain a deeper understanding of how Polkadot's flexible tech stack lets you easily build without limits.</p>
```

Comment:
Just nit-picking here, but I am curious about what you think of the following CTA:

```suggestion
                <p>Unlock the potential of Polkadot’s multi-chain architecture. Explore everything from an overview of the Polkadot Protocol to in-depth insights on parachain and system chains implementations, and discover how Polkadot’s flexible tech stack empowers you to build without limits.</p>
```